.github/workflows/deploy.yaml

```diff
@@ -24,6 +24,20 @@ env:
   IMAGE_NAME: ${{ github.repository }}
 
 jobs:
+  lint:
+    name: Lint Markdown with Vale
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - name: Run Vale linting
+        uses: errata-ai/vale-action@v2.1.1
+        with:
+          debug: true
+          fail_on_error: true
+          reporter: github-check
+
```

Comment:
I feel this to be defined in `.github/workflow/pr-request.yaml` instead. 
(Anyway, I just noticed the name of `pr-request.yaml` is completely weird 😮‍💨 )
[pr-request.yaml](https://github.com/GoogleCloudPlatform/khi/blob/main/.github/workflows/pr-request.yaml)